### ISSUE TITLE
README is wrong

### ISSUE THREAD

**Issue by yurishkuro** [uber/jaeger-client-node#23]
Instructions in the readme are broken.
- submodule does not init
- tests don't work
- I believe we need specific versions of Node and npm

```
$ cd jaeger-client-node
$ git submodule init update
error: pathspec 'update' did not match any file(s) known to git.
```

so fix it by 

```
$ git submodule init
$ git submodule update

# now try running tests
$ npm test
...
sh: flow: command not found
sh: eslint: command not found
...
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```

**Comment by frankgreco** [comment]
I also experienced this. My workout was running all commands as root:

```sh
$ sudo su
# ...
```

**Comment by yurishkuro** [comment]
I think this was resolved in 7804b0ef6c8c0cec15e8238292d954718438b09f
